style-sheets/primevue-components-overrides.scss

```diff
@@ -574,28 +552,49 @@ ul#nav li.active a {
     border-bottom-left-radius: 0.5rem;
 }
 
-.#{variables.$primevue-prefix}-dialog-footer {
+.#{variables.$primevue-prefix}-dialog
+    .#{variables.$primevue-prefix}-dialog-footer {
+    border-top: none;
     padding-bottom: 64px !important;
```

Comment:
Same as above, maybe we could change it to rem instead of px.

Reply:
thanks Nick!!